src/style.css

```diff
@@ -1 +1,78 @@
-/* Styles go here */
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  display: block;
+  position: relative;
+  top: 10px;
+  height: 40px;
+  width: 40px;
+}
+
+.icon {
+  height: 100%;
+  width: 100%;
+}
+
+.nav {
+  display: flex;
+}
+
+.nav__list {
+  display: flex;
+  list-style-type: none;
+  padding: 0;
+  margin: 0;
+}
+
+.nav__item {
+  margin-left: 20px;
+}
+
+.nav__item:first-child {
+  margin-left: 0;
+}
+
+.nav__link {
+  position: relative;
+  display: flex;
+  align-items: center;
+  height: 60px;
```

Comment:
do not hard code height - use padding instead

Reply:
Could you please explain why height 60 px here is hard code?
As far as I understand if I use here just padding-top: 30px and padding-bottom 30px - the height of the nav will be much larger than it should be

Reply:
UPD: I used padding 23px + 0 to fix it and it works!